### DISCUSSION TITLE
Gte Forwarding History Failed Error Code 502

### DISCUSSION THREAD

**Discussion by clarketowson** [btcpayserver/btcpayserver#5523] [Technical Support]
Hi Everyone,

I have BTCPay Server v1.3.5.0 running at the moment which I have not yet upgraded to the latest version.  I'm likely to upgrade in the near future.  The issue I'm having at the moment is I receive this **Get Forwarding History Failed** message when I login to my Ride the Lightning instance.  It seems to be an **nginx error 502 bad gateway**.  I think it might be caused by a too short timeout in nginx as it tries to get all the forwarding history and routing history logs which are extensive on my server.  If I restart the server sometimes the error message doesn't occur and i'm able to see the forwarding and routing history.

![B1](https://github.com/btcpayserver/btcpayserver/assets/69452308/d047910e-e355-46b3-a718-a22d3173fdbc)

![B2](https://github.com/btcpayserver/btcpayserver/assets/69452308/f8acd702-c172-4599-b158-c38943910883)

Any help with how I can resolve this issue would be very much appreciated.

Kind Regards,

Clarke

Clarke Towson
CEO
INTJ Billing
 
m: +61 432 359 166
a: 7 Cullen Court Spotswood Victoria 3015 AUSTRALIA
w: https://intjbilling.com/
e: [clarketowson@intjbilling.com](mailto:clarketowson@intjbilling.com)

**Comment by dennisreimann** [comment]
There is a an [open issue on the RTL repo](https://github.com/Ride-The-Lightning/RTL/issues/1031) about this. In general you should upgrade and see if that fixes it, because the version you are running is almost two years old.

**Reply by clarketowson** [reply]
Thank you for this info much appreciated.
